spec/support/contexts/response_tree.rb

```diff
@@ -29,4 +29,27 @@ def expect_children(node, types, qing_ids, values = nil)
     child_values = children.map { |child| child.is_a?(Answer) ? child.casted_value : nil }
     expect(child_values).to eq values
   end
+
+  def web_answer_hash(q_id, values, relevant: "true", destroy: nil)
```

Comment:
Comments on these methods would be nice I think as they are kind of peculiar?

Reply:
will add these